system/jlib/jlog.cpp

```diff
@@ -2324,6 +2324,77 @@ MODULE_EXIT()
     thePassAllFilter = NULL;
 }
 
+#ifdef _CONTAINERIZED
+
+#define LOGFIELDS_ATT "@fields"
+#define LOGMSGDETAIL_ATT "@detail"
+#define LOGMSGAUDIENCES_ATT "@audiences"
+#define LOGMSGCLASSES_ATT "@classes"
+#define USELOGQUEUE_ATT "@useLogQueue"
+#define LOGQUEUELEN_ATT "@queueLen"
+#define LOGQUEUEDROP_ATT "@queueDrop"
+#define USESYSLOG_ATT "@enableSysLog"
+
+#ifdef _DEBUG
+  #define DEFAULT_USELOGQUEUE false
+#else
+  #define DEFAULT_USELOGQUEUE true
+#endif
+
+#define DEFAULT_QUEUELEN 512
+#define DEFAULT_QUEUEDROP 32
+#define DEFAULT_USE_SYSLOG false
```

Comment:
trivial: more modern to use e.g.:
```
static constexpr const char *logFieldsAtt = "@fields";
```

(and compiler can validate declaration and possibly optimize some uses)